examples/README.md

```diff
@@ -1,22 +1,60 @@
 # How to test RBD and CephFS plugins with Kubernetes 1.13
 
+## Deploying Ceph-CSI services
+
 Both `rbd` and `cephfs` directories contain `plugin-deploy.sh` and
 `plugin-teardown.sh` helper scripts.  You can use those to help you
 deploy/teardown RBACs, sidecar containers and the plugin in one go.
 By default, they look for the YAML manifests in
 `../../deploy/{rbd,cephfs}/kubernetes`.
 You can override this path by running `$ ./plugin-deploy.sh /path/to/my/manifests`.
 
+## Creating clusterID named secret for RBD based provisioning
+
+**NOTE:** This section is not required for cephfs based provisioning, and SHOULD
+be skipped.
+
+For RBD based provisioning, the CSI plugin requires configuration information
+regarding the Ceph cluster(s), that would host the RBD based block devices. This
+is provided by adding a per-cluster secret named `ceph-cluster-<cluster-id>`,
+using the provided [template](./rbd/template-ceph-cluster-ID-secret.yaml).
+
+Gather the following information from the Ceph cluster(s) of choice,
+
+* Client admin ID and key, that has privileges to perform CRUD operations on
+  the Ceph cluster and pools of choice
+  * Key is typically the output of, `ceph auth get-key client.admin` where
+    `admin` is the client Admin ID
+  * This is used to substitute `[admin|user]id` and `[admin|user]key` values in
+    the template file
```

Comment:
I'm not seeing a particular reason why the ceph-csi needs the admin key. Is it so that it can do `ceph osd lspools`?

Reply:
The reason was a simple carryover of what existed prior to this PR. The exact credentials and its permissions are being worked out in issue #270 when we also need to think about what rights are required for the various operations being performed (like `ceph osd lspools`).